atlasdb-rocksdb-tests/src/test/java/com/palantir/atlasdb/cleaner/RocksDbSweeperTest.java

```diff
@@ -27,10 +27,9 @@
 public class RocksDbSweeperTest extends AbstractSweeperTest {
     private File tempDir;
 
-    @Override
     @After
     public void tearDown() {
```

Comment:
I kept the tearDown() method name for the 3 or so tests that actually do teardown by manually cleaning up